packages/sqs/test/fakes/FakeConsumerErrorResolver.ts

```diff
@@ -1,15 +1,18 @@
 import { SqsConsumerErrorResolver } from '../../lib/errors/SqsConsumerErrorResolver'
 
 export class FakeConsumerErrorResolver extends SqsConsumerErrorResolver {
+  public errors: Error[]
   public handleErrorCallsCount: number
   constructor() {
     super()
 
     this.handleErrorCallsCount = 0
+    this.errors = []
   }
 
   public override processError(error: unknown) {
     this.handleErrorCallsCount++
+    this.errors.push(error as Error)
```

Comment:
🟢 Wondering if we should control the case of error not being of type Error

Reply:
I wouldn't do it until there is a real case of that being a problem, and that has never happened in my experience in any library so far